### DISCUSSION TITLE
Downtime when updating installation

### DISCUSSION THREAD

**Discussion by beastawakens** [plausible/analytics#1127] [Self-Hosted Support] [answered]
So, running Plausible via the recommended Docker compose setup, I've noticed that if I'm updating the image for the analytics container and re-running the docker-compose up, that it can take 15-45ish seconds for the server to start back up again.  That's suitably impressive, but obviously during that time, any events or data sent to the server are lost.
How do you mitigate against that for the cloud offering? Do you have a buffer/queue in front of the server to ensure no data loss?

Thanks

**Comment by ukutaht** [comment, marked as answer]
Currently we're in the same boat with the cloud offering. I'm aware it's not optimal but we've had bigger fish to fry up to this point. We'll get to it eventually :)
